pages/linux/objdump.md

```diff
@@ -0,0 +1,15 @@
+# objdump
+
+> View information on object files.
+
+- Display the file header information:
+
+`objdump -f {{binary}}`
+
+- Display the dis-assembled output of executable sections:
```

Comment:
"dis-assembled" or "decompiled"? I'm not sure which one is more accurate.

Reply:
'dis-assembled' is better because it actually shows the assembly output from the binary. decompiled usually means 'binary -> source code'. There are java and .net decompilers which do that.

Also, it's literally taken from the expanded flag of `-d` which was `--disassemble`